pages/download.html

```diff
@@ -1,54 +1,63 @@
 <#include "incl/macros.html">
-<#def TITLE>Coq <#CURRENTVERSION></#def>
+<#def TITLE>Install Coq</#def>
 <#include "incl/header.html">
 
 <div class="framework">
-<div class="frameworklabel">The current version: Coq <#CURRENTVERSION>
+<div class="frameworklabel">The Coq platform
 </div>
 <div class="frameworkcontent">
 
-<p>For downloads (in particular to get installers for Windows and Mac OS, the PDF
-  manual, or a tarball of the sources), please go to
-  <a href="<#CURRENTVERSIONURL>">the release page on GitHub</a>.</p>
+<p>Coq has a rich ecosystem of external packages (libraries and
+plugins) that extend it and make it more powerful.  The Coq platform
+provides an easy way to install Coq and a consistent set of packages
+on Windows, macOS and many Linux distributions.</p>
+
+<p>Beginners are encouraged to use one of the binary installers: we
+provide <a href="https://github.com/coq/coq/releases/latest">binary
+installers</a> for Windows and macOS as well as
+a <a href="https://snapcraft.io/coq-prover">Snap package</a>
+compatible with many Linux distributions.</p>
+
+<p>Experienced users are advised to run the scripts provided
+by <a href="https://github.com/coq/platform">the platform</a> to
+install from sources as this will allow them to further extend the
+resulting package set <a href="/opam-using.html">using opam</a>.</p>
 
 </div>
 <div class="frameworklinks">
 <ul>
-<li><a href="<#CURRENTVERSIONURL>">Stable release (<#CURRENTVERSION>)</a></li>
-<#ifdef UPCOMINGVERSION>
-<li><a href="<#UPCOMINGVERSIONURL>">Testing release (<#UPCOMINGVERSION>)</a></li>
-</#ifdef>
+<li><a href="https://github.com/coq/coq/releases/latest">Binary installers (Windows and macOS)</a></li>
+<li><a href="https://snapcraft.io/coq-prover">Snap package</a></li>
+<li><a href="https://github.com/coq/platform">Platform scripts</a>
 </ul>
 </div>
 </div>
 
+
 <div class="framework">
-<div class="frameworklabel">How to install Coq on various systems</div>
+<div class="frameworklabel">Other installation methods
+</div>
 <div class="frameworkcontent">
 
-<p>
-The Coq wiki contains three helpful pages detailing various ways of
-installing Coq for the three most standard operating systems:
-</p>
+<p>Coq is available
+through <a href="https://repology.org/metapackage/coq/versions">many
+package managers</a>, including most Linux distribution package
+managers.  However, in most cases, the available version is not
+the latest version and, more importantly, installing Coq this way
+doesn't allow extending it with external packages, so this
```

Comment:
Fortunately, that is not true. Users did not wait for Opam and Nix before they started to use external packages with Coq. First, the good old way (`make && sudo make install`) still works for a large range of libraries. Second, these external packages might just be part of the Linux distribution too. For example, Fedora users can simply type `dnf install flocq`, since Flocq is part of the main repository. Thus, I would rather write:

"More importantly, these distributions might provide only a fraction of all the external packages available for Coq, thus requiring some manual compilation and installation to extend it."